packages/notification-service/src/index.ts

```diff
@@ -1,11 +1,45 @@
+import '../load.envs.js';
+
+import { PrismaClient } from '@prisma/client';
 import type { Request, Response } from 'express';
 import express from 'express';
+import { Provider } from 'fuels';
+import { createPublicClient, http } from 'viem';
+import { foundry, sepolia } from 'viem/chains';
+
+import { handleNewEthBlock } from './utils/handleNewEthBlock';
+
+let NOTIFY_LOCK = false;
 
 const notificationServer = express();
 const port = 3005;
 
-notificationServer.get('/', (req: Request, res: Response) => {
-  res.send('hello');
+const isDev = process.env.ETH_CHAIN === 'foundry';
+
+const fuelProvider = new Provider(
+  isDev ? process.env.FUEL_PROVIDER_URL! : 'https://beta-4.fuel.network/graphql'
+);
+
+const ethPublicClient = createPublicClient({
+  chain: isDev ? foundry : sepolia,
+  transport: isDev
+    ? http('http://localhost:8545')
+    : http(
+        'https://eth-sepolia.g.alchemy.com/v2/v_2w-fv1Jg2R3TMKbT3N_p7zue7EeA9N'
+      ),
```

Comment:
On the docs it says that transport is just `transport: http()` why we need to grap config again here as it cames from the load.env.js?